src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+import lottery.Color;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
This method should return color, in our program color is enum, so please return propriate value here